src/battery_manager.cpp

```diff
@@ -175,7 +173,7 @@ int main(int argc, char **argv)
     diag_pub.publish(diag_msg);
 
     // Publish voice message if new
-    if (old_message != message)
+    if (old_message != message && message != "")
```

Comment:
How can message be ""? Can this ever happen?

Reply:
https://github.com/tue-robotics/battery_manager/blob/master/src/battery_manager.cpp#L123